docs/docs.trychroma.com/markdoc/content/docs/querying-collections/full-text-search.md

```diff
@@ -14,6 +14,15 @@ In order to filter on document contents, you must supply a `where_document` filt
 }
 ```
 
+You can also filter documents using a regex pattern, with the `$regex` and `$not_regex` keys:
```

Comment:
Asks: 
- Can we spell out "regular expressions" somewhere just for SEO purposes?
- Can we link to some documentation about regular expressions, such as some nice web UI that allows you to validate regular expressions or something?
- It would be useful to have one regular expression that someone could copy and paste to test more generally. I'm thinking maybe a regular expression for case-insensitive search.